src/stylesheets/components/_skipnav.scss

```diff
@@ -1,17 +1,15 @@
 .usa-skipnav {
   background: transparent;
-  color: $color-base;
   left: 0;
-  padding: 1rem 1.5rem;
+  padding: units(1) units(2);
   position: absolute;
-  top: -4.2rem;
+  top: -3.8rem; // skipnav link height
   transition: all 0.2s ease-in-out;
   z-index: 100;
```

Comment:
What's the variables/function for z-index?

Reply:
There isn't a function for that one yet either

Reply:
Updated in e6add3b.